### PR TITLE
Hided default arrows of input type number

### PR DESCRIPTION
**Pull request by MiguelP1989**

### CODE REVIEW

File: src/components/Currency/styled.ts [hunk 1]
```diff
@@ -100,6 +100,16 @@ const Wrapper = styled(FormFieldContainer)`
             outline: 0;
             width: calc(100% - 20px);
         }
+        /* Chrome, Safari, Edge, Opera */
+        input::-webkit-outer-spin-button,
+        input::-webkit-inner-spin-button {
```
Comment (harman052) [inline src/components/Currency/styled.ts:105]: Can we try some other approach? MDN discourages using these properties as these are non standard. https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-outer-spin-button

**Review by harman052 — CHANGES_REQUESTED** [review]

File: src/components/Currency/styled.ts (hunk 1 above)
Comment (MiguelP1989) [reply src/components/Currency/styled.ts:105]: Didn't see this coming!! I will see if there is another alternative! Thanks
We can remove also the type property "number".

Comment (harman052) [reply src/components/Currency/styled.ts:105]: I found this:

```<input type="text" inputmode="numeric" pattern="[0-9]*">```

from https://stackoverflow.com/questions/3790935/can-i-hide-the-html5-number-input-s-spin-box

Comment (MiguelP1989) [reply src/components/Currency/styled.ts:105]: Nice find!! ;)

File: src/components/Currency/hooks.ts [hunk 2]
```diff
@@ -39,7 +39,14 @@ function useHandlers({
         event.stopPropagation();
 
         const { value: nextValue } = event.target;
-        updateValuesHandler({ value: nextValue ? parseFloat(nextValue) : undefined, currency: currency.label });
+        const isNumber = /^\d*[\.\,]?\d*$/;
+
+        if (isNumber.test(nextValue)) {
+            updateValuesHandler({
+                value: nextValue ? nextValue : '',
```
Comment (harman052) [inline src/components/Currency/hooks.ts:46]: Your linter should have complaint here. We can simplify it as:

```suggestion
                value: nextValue || '',
```

**Review by harman052 — COMMENTED** [review]
One tiny comment.

File: src/components/Currency/hooks.ts (hunk 2 above)
Comment (MiguelP1989) [reply src/components/Currency/hooks.ts]: done ;)

**Review by harman052 — APPROVED** [review]

**Pull request merged**